### PR TITLE
Tour persistence

### PR DESCRIPTION
**Pull request by jamiander**
Tour persistence.  Added missing div.  fixed bug with updating user.

### CODE REVIEW

**Comment by jamiander** [comment]
There is a function in UserDetailsServiceImplementation to get the current user.  The function is called in the TourController.

File: src/main/resources/templates/gardens/list.html [hunk 1]
```diff
@@ -60,28 +60,27 @@ <h4 class="garden-subtitle">Here are the water gardens that are featured on our
         <th:block th:if="${title.equals('Garden List')}">
             <p>Displaying All Gardens</p>
         </th:block>
-        <!-- hide link if on main gardens/list/ (only show when on search)-->
+    </div>
 
-        <hr>
-        <div th:fragment="garden-card" class="container-fluid">
```
Comment (hstrodtman) [inline src/main/resources/templates/gardens/list.html:66]: It looks like this line being changed is messing with the categories functionality on the home page.

**Review by hstrodtman — APPROVED** [review]
Good work!

**Review by MulesKC — APPROVED** [review]
What an awesome feature!

File: src/main/resources/templates/gardens/list.html (hunk 1 above)
Comment (MulesKC) [reply src/main/resources/templates/gardens/list.html:66]: Should I set up the garden-card in fragments.html for the front page features? If so do you want me to have these cards pull from there too?

**Review by MulesKC — APPROVED** [review]
Awesome feature, I love it! Just had that question about feature-view in there.

**Pull request merged**